projects/storefrontlib/src/layout/direction/config/default-direction.config.ts

```diff
@@ -0,0 +1,14 @@
+import { DirectionConfig } from './direction.config';
+
+export const defaultDirectionConfig: DirectionConfig = {
+  direction: {
+    detect: true,
+
+    // we hold back with default direction during minor release
+    // default: DirectionMode.LTR,
```

Comment:
Dropping the default config for the `default` property doesn't suffice to avoid breaking change. Please note that the method `initialize()` manipulates the `dir` anyway (regardless the `default` propery and event the `detect` property). 

Say, our customer have alrady hardcoded `<html dir="rtl">`. `initialize()` will either change it or clear it on app start. That is why the body of the `subscribe()` in `initialize()` should be feature-level-guarded IMO.

I can see the downside will be the `FeaturesConfigService` constructor dependency, but I don't mind it that much, as we can remove it in 3.0